src/sys/select.rs

```diff
@@ -227,6 +300,29 @@ mod tests {
         assert!(!fd_set.contains(r2));
     }
 
+    #[test]
+    #[cfg_attr(any(target_arch = "powerpc", target_arch = "mips"), ignore)]
```

Comment:
If you're going to ignore the test on those two arches, you should explain why in a comment.

Reply:
Ah! I forgot to drop that line when I cargo-culted this test. Thanks for spotting it!